### PR TITLE
issue/316 Component header background

### PR DESCRIPTION
**Pull request by guywillis**
Resolves https://github.com/adaptlearning/adapt-contrib-vanilla/issues/316

* Adds component header background colour as an enhancement

### CODE REVIEW

File: less/project/theme-extras.less [hunk 1]
```diff
@@ -196,6 +196,52 @@
...
+  .pagelevelprogress {
+    &__indicator {
+      border-color: @@color-inverted;
+    }
+    &__indicator-inner {
+      background-color: @@color;
+    }
+    &__indicator-bar {
+      background-color: @@color-inverted;
+    }
```
Comment (eleanor-heath) [inline less/project/theme-extras.less:239]:
```suggestion
    &__indicator {
      border-color: @@color-inverted;
      
      &-bar {
        background-color: @@color-inverted;
      }
    
      &-inner {
        background-color: @@color;
      }
    }
```

**Review by eleanor-heath — APPROVED** [review]

File: less/project/theme-extras.less (hunk 1 above)
Comment (guywillis) [reply less/project/theme-extras.less:239]: Are you happy to leave this as is as it follows the `bg-color` definition in Vanilla's `theme-extras` https://github.com/adaptlearning/adapt-contrib-vanilla/blob/master/less/project/theme-extras.less#L228

Comment (eleanor-heath) [reply less/project/theme-extras.less:239]: yes of course no problem

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**